doc/source/actions.rst

```diff
@@ -73,6 +73,33 @@ parameter:
 Once the backporting pull request is closed or merged, Mergify will
 automatically delete the backport head branch that it created.
 
+copy
+====
+
+The `copy` action just create a copy of the pull request as soon as the
+condition match.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 1 1 1 2
+
+   * - Key Name
+     - Value Type
+     - Default
+     - Value Description
+   * - ``branches``
+     - array of string
+     - ``[]``
+     - The list of branches the pull request should be copied to.
+   * - ``regexes``
+     - array of string
+     - ``[]``
+     - The list of regexes to find branches the pull request should be copied to.
+
+Once the copy pull request is closed or merged, Mergify will
```

Comment:
I'd say `new pull request` rather than `copy pull request`. I find it confusing since I'm not sure you're talking about the one that is the source of the new one.

> Once the new pull requests are closed or merged, Mergify will...